models/token.js

```diff
@@ -9,12 +9,9 @@ const MODEL = {
         .integer()
         .positive(),
 
-    uuid: Joi
+    value: Joi
         .string()
```

Comment:
Should limit it to:

* The character set we would allow
* The length of the value if it's fixed-length